modules/thunder_updater/updates.yml

```diff
@@ -71,3 +71,6 @@ v1.1:
         '#title': 'Add image paragraph to liveblog posts'
         '#description': "<p>Instead of adding images in the ckeditor, this allows you to place an image as an embedded media at the top of the post.</p>"
         '#description_failed': '<p><strong>There was a problem during the update.</strong> Go to "Structure" -> "Liveblog posts" -> "Manage fields" and click on "Edit" next to the field "Embed media". Here you can select the image paragraph.</p>'
+    v1_1__thunder_amp:
+        '#title': 'Use AMP (Accelerated Mobile Pages by Google)'
+        '#description': '<p>This module optimises your articles for AMP, so that they load very fast on mobile devices and can be viewed directly on Google.</p><p>To use it, go to "Appearance" and activate the `Thunder AMP` theme.</p><p>ATTENTION: If you are not using a composer workflow, due to licensing issues, you first have to download the AMP library. With composer installed execute the command `composer require lullabot/amp` in the docroot of your installation.</p>'
```

Comment:
Can you please add links to relevant pages. For example:
- admin/appearance - "Appearance"
- admin/modules
- and link to library: https://github.com/Lullabot/amp-library

You can find already existing examples, fe. v1_1__shariff